classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_students(student)
+    @students.push(student)
```

Comment:
- [ ] To implement the `has-many` of the relationship, when adding a student to a classroom it also sets the classroom for the student, for example: 
```ruby
  def add_student(student)
    @students << student
    student.classroom = self
  end
```
Please kindly fix that.